src/code-util.jl

```diff
@@ -189,43 +199,25 @@ function extract_problems(results)
     rel_code_key = "/:rel/:catalog/:diagnostic/:code/Int64/String"
     rel_line_key = "/:rel/:catalog/:diagnostic/:range/:start/:line/Int64/Int64/Int64"
     rel_severity_key = "/:rel/:catalog/:diagnostic/:severity/Int64/String"
+    rel_message_key = "/:rel/:catalog/:diagnostic/:message/Int64/String"
 
     if !haskey(results, rel_code_key)
         return problems
     end
 
-    # [index, code]
-    problem_codes = results[rel_code_key]
+    indices = results[rel_code_key][1]
```

Comment:
It's used once, on the line below. It wasn't really being used as 'indices', though, just to provide a count of indices. I changed the following line so that it was. This is a mildly fundamental change since instead of counting the indices used from 1 to length, it now extracts them from the diagnostics tree.